### PR TITLE
Display `NrOfPoints` GSPOE-19

### LINKED ISSUE #912: NrOfPoints not displayed in PDF extract
For PLR that are points the number of points is not displayed in the pdf extract

in this extract we see on p.9 29 points on the parcel. Here I would expect pyramid_oereb to output this number in the column "Anteil".
[20191009111416_extract.pdf](https://github.com/openoereb/pyramid_oereb/files/3706516/20191009111416_extract.pdf)

### PR DESCRIPTION
**Pull request by marionb**
Fix #912 
What this PR does:
- Add `NrOfPoints` in the the legend json
- Format `NrOfPoints` to a string in the print json
- Add a variable `NrOfPoints` that is accessible in the print  templates (see PR https://github.com/openoereb/pyramid_oereb_mfp/pull/40)
- Add test data to be able to test this in the pdf extract.

To test and get an actual result an adaption in the templates was necessary -> https://github.com/openoereb/pyramid_oereb_mfp/pull/40

### CODE REVIEW

**Comment by pfirpfel** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- pyramid_oereb/contrib/print_proxy/mapfish_print.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/pyramid_oereb/pyramid_oereb/pullRequest?prid=4622369&bid=15500188)

**Comment by jwkaltz** [comment]
@vvmruder were you able to test this? A report from Geoinfo / Max makes me uncertain that this PR fixed the problem entirely. We are transmitting NrOfPoints now to the print and showing it in the PDF, but I am not sure we are missing something regarding summing up individual NrOfPoints?

**Comment by peterschaer** [comment]
I was finally able to test this with 1.7.0. NrOfPoints is now displayed in the pdf extract.

points_pdf

But the count is always 1 (instead of 29 in this case). The JSON oder XML has 29 point objects:

[CH873580374655.zip](https://github.com/openoereb/pyramid_oereb/files/4160135/CH873580374655.zip)

But print_proxy doesn't sum up correctly. Here is the getspec-Output:

[CH873580374655.zip](https://github.com/openoereb/pyramid_oereb/files/4160148/CH873580374655.zip)

The summing up doesn't seem to be correct.
